packages/xpetra/src/CMakeLists.txt

```diff
@@ -37,6 +116,9 @@ IF (${PACKAGE_NAME}_ENABLE_Epetra)
     )
 ENDIF()
 
+MESSAGE("JHU: CMAKE_CURRENT_SOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
I think there's a way to mark a PR as a "draft."

Reply:
Woops, this was a debugging message, it should be deleted.